Square.rb

```diff
@@ -3,10 +3,12 @@ class Square
   attr_reader :side
   attr_accessor :border
   
+  # Needed to create a new Square object
   def initialize(side)
     @side = side
   end
 
+  # Used to compute area quickly
```

Comment:
Thanks @githubteacher! I'll merge this now that you've commented and haven't found any issues!